src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing.mdx

```diff
@@ -17,9 +17,15 @@ For how to find billing and usage information in the UI, see [Billing-related UI
 
 ## Data ingest calculation [#usage-calculation]
 
-In the context of the [New Relic One pricing model](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing), “ingested data” refers to the data saved for your New Relic organization (your set of accounts sharing the same billing) after we apply various data trimming and data transformation rules. In other words, it’s not the amount of raw data sent to New Relic, but the size of the data that actually ends up being stored. 
+For the [New Relic One pricing model](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing), “ingested data” refers to the data saved for your New Relic organization after we apply various data trimming and data transformation rules. In other words, it’s not the amount of raw data sent to New Relic, but the size of the data that actually ends up being stored. 
 
-To view and manage your usage, go to the [usage UI](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/billing-ui).
+Here are some notes about New Relic features and data that **do not** count towards ingest and billing:  
+  * [Basic alerting functionality](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/alerts-concepts-workflow) doesn't count towards billing. 
+  * [Applied intelligence](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/introduction-applied-intelligence): data reported from third-party services counts towards billing but internally generated data (for example, `NrIncidentEvent` data) does not. 
```

Comment:
@zuluecho9, I would go for more consistency by ditching the colon: "Data reported in [Applied intelligence](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/introduction-applied-intelligence) from third-party services counts towards billing but internally generated data (for example, `NrIncidentEvent` data) does not."